proposals/nnnn-sdl-javascript-manager-layer.md

```diff
@@ -0,0 +1,96 @@
+# JavaScript Suite SDL Manager
+
+* Proposal: [SDL-NNNN](NNNN-js-suite-managers.md)
+* Author: [Chris Rokita](https://github.com/crokita)
+* Status: **Awaiting review**
+* Impacted Platforms: JavaScript Suite
+
+## Introduction
+
+In a similar vein to [this proposal](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0171-android-manager-apis.md), the goal is to introduce a manager layer to the JavaScript suite, closely matching the Android and iOS manager APIs. These manager classes are designed to abstract out complex flows and state for common tasks, such as file handling and displaying information to the screen. 
```

Comment:
```suggestion
In a similar vein to [this proposal](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0171-android-manager-apis.md), the goal is to introduce a manager layer to the JavaScript Suite, closely matching the Java Suite and iOS manager APIs. These manager classes are designed to abstract out complex flows and states for common tasks, such as file handling and displaying information to the screen. 
```
Please make sure you update the PR description with these changes as well, thanks!